README.md

```diff
@@ -253,8 +253,9 @@ component's "real" props. In this example lets use `_testID`.
 ```js
 let Greeting = props => <div>hello <strong _testID='name'>{props.name}</strong></div>;
 
-$(Greeting).render()
-  .prop({ name: 'Betty' })
+$(<Greeting />)
```

Comment:
thanks, tho I think the thing would be to do `$(<Greeting name='Betty' />)` for the example as its a bit simpler